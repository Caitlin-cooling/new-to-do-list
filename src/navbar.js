import './global.js';
import { Navbar, Nav, NavItem } from "react-bootstrap";
import { BrowserRouter as Router, Route, Link } from 'react-router-dom';

export class NavBar extends React.Component {
  render() {
    return <Router>
      <Navbar>
        <Nav>
          <Navbar.Header>
            <Navbar.Brand>
            <Link className="home-button" to="/">
              Do it
            </Link>
            </Navbar.Brand>
          </Navbar.Header>
          <NavItem>
            <Link className="new-button" to="/new">
              Create new
            </Link>
          </NavItem>
        </Nav>
      </Navbar>
    </Router>
  }
}
