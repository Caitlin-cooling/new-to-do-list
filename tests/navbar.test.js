import React from 'react';
import renderer from 'react-test-renderer';
import { Navbar, Nav, NavItem } from "react-bootstrap";
import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
import { NavBar } from '../src/navbar.js';

test('navbar is rendered', () => {
  const component = renderer.create(
    <NavBar/>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
