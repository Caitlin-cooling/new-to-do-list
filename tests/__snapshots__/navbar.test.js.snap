// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar is rendered 1`] = `
<nav
  className="navbar navbar-default"
>
  <div
    className="container"
  >
    <ul
      className="nav navbar-nav"
      role={null}
    >
      <div
        className="navbar-header"
        onSelect={null}
      >
        <a
          className="home-button navbar-brand"
          href="/"
          onClick={[Function]}
        >
          Do it
        </a>
      </div>
      <li
        className=""
        role="presentation"
      >
        <a
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
        >
          <a
            className="new-button"
            href="/new"
            onClick={[Function]}
          >
            Create new
          </a>
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
